### PR TITLE
Collators v8 - session / nimbus keys 

### PR DESCRIPTION
**Pull request by artkaseman**
Adding in Alberto's collator changes for v8, primarily session keys.  

removed additional peers, reverted back to the default values.

### CODE REVIEW

File: node-operators/networks/collator.md [hunk 1]
```diff
@@ -82,3 +89,77 @@ The following table presents some of the timings in regards to different actions
 !!! note 
     The values presented in the previous table are subject to change in future releases.
 
+## Session Keys
+
+With the release of [Moonbase Alpha v8](/networks/testnet/), collators will sign blocks using an author ID, which is basically a [session key](https://wiki.polkadot.network/docs/en/learn-keys#session-keys). To match the Substrate standard, Moonbeam collator's session keys are [SR25519](https://wiki.polkadot.network/docs/en/learn-keys#what-is-sr25519-and-where-did-it-come-from). This guide will show you how you can create/rotate your session keys associated to your collator node.
+
+First, make sure you are running your collator node with the following flags. TODO: Art make sure the flags are included in the collator's code snippet, also. If there are some flags that add some security concerns, it might be good to point them out.
```
Comment (eshaben) [inline node-operators/networks/collator.md:96]: @artkaseman looks like some flags need to be added here

**Review by eshaben — CHANGES_REQUESTED** [review]
it's weird that all the waffle and mars content is being deleted in this PR but when I went through each commit, no one ever deletes it 🤔 

but looks like theres one TODO for you in here @artkaseman

File: node-operators/networks/collator.md (hunk 1 above)
Comment (artkaseman) [reply node-operators/networks/collator.md]: fixed

**Review by eshaben — APPROVED** [review]

**Comment by JoshOrndorff** [comment]
> It's weird that all the waffle and mars content is being deleted in this PR

It's not that Art deleted it. It's that he forked off from `master` before your waffle mars PR was merged in. So it just wasn't in the history of this PR branch. It should still be on master now that it is merged.

In the future you can `git merge master` on your PR branch and those red diff lines will go away. Can be nice for extra peace of mind.

**Comment by eshaben** [comment]
I tried to merge master and it kept saying that it was already up to date... I ended up cherry picking the commits back in

**Pull request merged**